public/index.html

```diff
@@ -93,148 +93,169 @@
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-22" class="img-responsive" src="" alt="">
-                        </a>
-                    </div>
-                    <div class="col-lg-3 col-centered thumb">
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-23" class="img-responsive" src="" alt="">
-                        </a>
-                    </div>
-                    <div class="col-lg-3 col-centered thumb">
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-24" class="img-responsive" src="" alt="">
-                        </a>
-                    </div>
-                </div>
-                <div class="row-centered">
-                    <div class="col-lg-3 col-centered thumb">
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-19" class="img-responsive" src="" alt="">
-                        </a>
-                    </div>
-                    <div class="col-lg-3 col-centered thumb">
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-6" class="img-responsive" src="" alt="">
-                        </a>
-                    </div>
-                    <div class="col-lg-3 col-centered thumb">
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-7" class="img-responsive" src="" alt="">
-                        </a>
-                    </div>
-                    <div class="col-lg-3 col-centered thumb">
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-8" class="img-responsive" src="" alt="">
-                        </a>
-                    </div>
-                    <div class="col-lg-3 col-centered thumb">
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-9" class="img-responsive" src="" alt="">
-                        </a>
-                    </div>
+                <div class="col-lg-7">
+                    <div class="row plotbox" id="neighbourpca"></div>
                 </div>
-                <div class="row-centered">
-                    <div class="col-lg-3 col-centered thumb">
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-18" class="img-responsive" src="" alt="">
-                        </a>
-                    </div>
-                    <div class="col-lg-3 col-centered thumb">
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-5" class="img-responsive" src="" alt="">
-                        </a>
+                <div class="col-lg-5 gallery">
+                    <div class="row-centered">
+                        <div class="col-lg-3 col-centered selected-image-display">
```

Comment:
Those are classes are built into Bootstrap that make the task of dividing your screen into x number of columns much easier. It works by splitting the screen up into 12 equal size columns, then you specify how wide you'd like your columns to be.. eg two ``col-lg-6`` will give you two equal columns. The number needs to add up to 12.

See these examples on the Bootstrap website: http://getbootstrap.com/examples/grid/